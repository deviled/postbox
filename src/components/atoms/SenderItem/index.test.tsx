import React from 'react';
import { render } from '__tests__/utils';
import { SenderItem } from '.';

describe('SenderItem.tsx', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<SenderItem name="" count={10} />);

    expect(baseElement).toMatchSnapshot();
  });
});
