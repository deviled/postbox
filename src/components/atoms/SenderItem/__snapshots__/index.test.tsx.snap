// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SenderItem.tsx should match snapshot 1`] = `
.c0 {
  padding: 16px;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  font-size: 1rem;
  margin-bottom: 16px;
  border: 1px solid lightgray;
  background-color: initial;
}

.c1 {
  padding: 4px 8px;
  border-radius: 50%;
  margin-left: 8px;
  background: lightgray;
}

<body>
  <div>
    <button
      class="c0"
    >
      
      <span
        class="c1"
      >
        10
      </span>
    </button>
  </div>
</body>
`;
